woocommerce-referralcandy.php

```diff
@@ -26,7 +26,7 @@
     die('Direct access is prohibited.');
 }
 
-if (in_array('woocommerce/woocommerce.php', apply_filters('active_plugins', get_option('active_plugins')))) {
+if (preg_grep("/\/woocommerce.php/", apply_filters('active_plugins', get_option('active_plugins'))) !== null) {
```

Comment:
`preg_grep("/\/woocommerce.php$/")` ? to make it more strict?

Reply:
ah yeah, good catch! give me a sec. 👍